src/main/java/school/hei/haapi/endpoint/rest/controller/FeeController.java

```diff
@@ -148,4 +151,9 @@ public FeeTemplate createOrUpdateFeeTemplate(
     return feeTemplateMapper.toRest(
         feeTemplateService.createOrUpdateFeeTemplate(feeTemplateMapper.toDomain(feeType)));
   }
+
+  @GetMapping("/fees/projection")
+  public File getUnpaidFeesProjection() {
```

Comment:
use Resource, also add produces IMAGE_PNG or whatever the format you use.
do not forget to write api spec for this endpoint;
Last but not least, do not forget to properly configure endpoint security

Reply:
done